Tasks/AzureRmWebAppDeployment/azurermwebappdeployment.ts

```diff
@@ -45,6 +45,17 @@ async function run() {
         var dockerNamespace = tl.getInput('DockerNamespace', false);
         var isDeploymentSuccess: boolean = true;
         var tempPackagePath = null;
+        var inputAppSettings = tl.getInput('AppSettings', false);
+        var imageSource = tl.getInput('ImageSource', false);
+        var startupCommand = tl.getInput('StartupCommand', false);
+        var isLinuxWebApp: boolean = webAppKind && webAppKind.indexOf("linux") >= 0;
+        var runtimeStack = "";
+        var linuxWebDeployPkg = "";
+
+        if (isLinuxWebApp && imageSource.indexOf("Builtin") >= 0) {
```

Comment:
imageSource can be null

Reply:
You are checking that later also. better to create another variable call isBuiltInLinuxWebAPp